### ISSUE TITLE
the command surface dgpu set on doesn't actually turn on the gpu.

### ISSUE THREAD

**Issue by BalenCPP** [linux-surface/surface-control#6]
So I have the linux-surface 5.5.4-1 installed, along with nvidia-dkms. I use surface-control to turn on the dGPU but when I try to switch to the dgpu using optimus-manager, it doesn't work. I thought that maybe the dGPU isn't on and so when I do lspci | grep 'VGA', it only prints `
00:02.0 VGA compatible controller: Intel Corporation UHD Graphics 620 (rev 07)
00:13.0 Non-VGA unclassified device: Intel Corporation Sunrise Point-LP Integrated Sensor Hub (rev 21)
`

**Comment by BalenCPP** [comment]
So I took a look at the code and apparently all it does is write on to /sys/devices/platform/MSHW0153:00/dgpu_power so I am guessing this is some issue with the kernel or my installation(reinstalled from scratch multiple times)?

**Comment by qzed** [comment]
> all it does is write on to /sys/devices/platform/MSHW0153:00/dgpu_power

That is correct. So if the `surface` tool doesn't give you any error the files is present which means that the driver is loaded. Is there anything in the dmesg log when you turn it on/off? Can you verify that this is happening without invoking any other programs that may change the dGPU power? I have no idea what optimus manager does exactly and haven't tested it, so if it tries to manage the power state by itself you might run into conflicts.

Also: Moving this to the surface-aggregator repo as it reads like this is either a driver or configuration issue (the surface control tool is just a command line front-end for the sysfs files provided by that module).
